### PR TITLE
Made Login.svelte,Signup.svelte, imported a font in App.svelte, made …

### PR DESCRIPTION
**Pull request by prchS**
Created the following : 

- Login.svelte
- Signup.svelte
- src/Images folder along with 3 images used, Facebook.png, google.png, linkedin.png

Made changes to the following:

- Header.svelte- Just added functionality to the hyperlink to take you to the respective page - (Login)
- Added/Imported a font into App.svelte
- Made changes to homepage.svelte  - added On click functions to the buttons to take them to the respective pages.

NOTE: Have to fix functioning of header - wont show up on either of the pages; goal for next push!

### CODE REVIEW

**Comment by suvanshdutt** [comment]
I've read it, all good can you resolve the conflict? Also extracting the route button on click to its own function in the script is a better idea.

**Comment by prchS** [comment]
Resolved the conflict

File: package.json
Comment (suvanshdutt) [inline package.json:1]: This dependency should not be removed

Comment (prchS) [reply package.json:1]: yea weird, not intentionally removed. Maybe something went wrong

Comment (suvanshdutt) [reply package.json:1]: Still is replacing the adapter-auto here.

**Review by suvanshdutt — APPROVED** [review]
I believe this is good now to merge after that fix.

**Pull request merged**